packages/jsii-pacmak/test/expected.jsii-calc/java/src/main/java/software/amazon/jsii/tests/calculator/DeprecatedClass.java

```diff
@@ -19,6 +19,8 @@ protected DeprecatedClass(final software.amazon.jsii.JsiiObject.InitializationMo
 
     /**
      * @deprecated this constructor is "just" okay
+     * @param readonlyString This parameter is required.
+     * @param mutableNumber
```

Comment:
shouldn't this say "the value to be set"?

Reply:
No, per previous comment from @bmaizels, we only want that description for params on builder setters, not constructors/class methods.